Makefile.in

```diff
@@ -140,7 +140,7 @@ BC_NUM_KARATSUBA_LEN = %%KARATSUBA_LEN%%
 CPPFLAGS1 = -D$(BC_ENABLED_NAME)=$(BC_ENABLED) -D$(DC_ENABLED_NAME)=$(DC_ENABLED)
 CPPFLAGS2 = $(CPPFLAGS1) -I./include/ -DVERSION=$(VERSION) %%LONG_BIT_DEFINE%%
 CPPFLAGS3 = $(CPPFLAGS2) -DEXECPREFIX=$(EXEC_PREFIX) -DMAINEXEC=$(MAIN_EXEC)
-CPPFLAGS4 = $(CPPFLAGS3) -D_POSIX_C_SOURCE=200809L -D_XOPEN_SOURCE=700
+CPPFLAGS4 = $(CPPFLAGS3) -D_POSIX_C_SOURCE=200112L -D_XOPEN_SOURCE=600
 CPPFLAGS5 = $(CPPFLAGS4) -DBC_NUM_KARATSUBA_LEN=$(BC_NUM_KARATSUBA_LEN)
```

Comment:
I do not like moving to POSIX 2001. It seems unnecessary. For example, `toybox` requires POSIX 2008, and it is plenty old. Can you make a case for doing switching to POSIX 2001?

Reply:
If you know it builds as 2001, it should also conform to 2008 standard, "unless otherwise noted" which isn't the case here. I don't have a *good* reason to say "you should downgrade" except that it's a benign change for any platforms you're already supporting.

Reply:
I can't find any POSIX 2008-specific stuff, so I will accept the downgrade.